tendrl/gluster_integration/manager/__init__.py

```diff
@@ -34,10 +34,12 @@ def main():
     NS.state_sync_thread = sds_sync.GlusterIntegrationSdsSyncStateThread()
 
     NS.node_context.save()
-    NS.tendrl_context.save()
-    NS.gluster_integration.definitions.save()
-    NS.gluster_integration.config.save()
+    NS.gluster.tendrl_context.save()
```

Comment:
This problem happens even with `node-agent` if re-started and no create/import executed after that. Even if earlier we had ran create/import and then the `integration_id` was populated. But after re-start again `tendrl_context.integration_id` goes blank.